### PR TITLE
Support java 11

### PR DESCRIPTION
**Pull request by stanlyDoge**

### CODE REVIEW

File: .travis/build.sh [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
-echo "Run Findbugs ..."
-make findbugs
+# The first segment of the version number is '1' for releases < 9; then '9', '10', '11', ...
+JAVA_MAJOR_VERSION=$(java -version 2>&1 | sed -E -n 's/.* version "([0-9]*).*$/\1/p')
+if [ ${JAVA_MAJOR_VERSION} -gt 1 ] ; then
+  export JAVA_VERSION=${JAVA_MAJOR_VERSION}
+fi
+
+if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
+  # sme parts of the workflow should be done only one on the main build which is currently Java 8
```
Comment (tombentley) [inline .travis/build.sh]:
```suggestion
  # some parts of the workflow should be done only on the main build which is currently Java 8
```

File: src/test/java/io/strimzi/kafka/bridge/amqp/AmqpSinkBridgeEndpointMockTest.java [hunk 2]
```diff
@@ -129,7 +130,7 @@ protected ProtonSender mockSender(ProtonQoS qos, String remoteAddress) {
         when(mockRecords.size()).thenReturn(1);
         // recordAt should not be called by the endpoint
         // but just to be sure...
-        when(mockRecords.recordAt(anyInt())).thenThrow(Exception.class);
+        when(mockRecords.recordAt(anyInt())).thenThrow(UncheckedIOException.class);
```
Comment (tombentley) [inline src/test/java/io/strimzi/kafka/bridge/amqp/AmqpSinkBridgeEndpointMockTest.java]: Why this change?

Comment (stanlyDoge) [reply src/test/java/io/strimzi/kafka/bridge/amqp/AmqpSinkBridgeEndpointMockTest.java]: The mockito was complaining about `Checked Exception`.
